API : POST /api/comment

moaikang opened the pull request:
comment를 추가하는 API를 생성하였다.

### Issue Number
Close #87

### 새로운 기능
- Comment Create.

### 작업 유형
- [x] 신규 기능 추가
- [ ] 버그 수정
- [ ] 리펙토링
- [ ] 문서 업데이트

### 체크리스트
- [x] Merge 하는 브랜치가 올바른가?
- [x] 코딩컨벤션을 준수하는가?
- [x] PR과 관련없는 변경사항이 없는가?
- [x] 내 코드에 대한 자기 검토가 되었는가?
- [ ] 변경사항이 효과적이거나 동작이 작동한다는 것을 보증하는 테스트를 추가하였는가?
- [ ] 새로운 테스트와 기존의 테스트가 변경사항에 대해 만족하는가?

Commit:
feat    : :zap:️ API : POST /api/comment

comment를 추가하는 API를 생성하였다.

maong0927 commented on web/server/src/routers/comment.ts:
```diff
@@ -1,5 +1,18 @@
+  try {
+    const { body } = req;
+    const result = await CommentService.create(body);
+    res.status(200).json({ status: 200, message: "OK", data: result })
```
status(200)으로 상태 코드를 설정한 뒤  body에서 다시 200 코드를 설정해서 보내야 할까욤 ???

moaikang replied:
회의 결과에 따라 수정했습니다!👍

Commit:
refactor: :rainbow: Comment API refactor

Comment Post API를 response 컨벤션 정한대로 리팩토링했다.
또한, 변수의 이름을 result -> newComment로 더욱 의미있게 바꾸었다.

Commit:
refactor: :rainbow: comment API resposne refactor

comment APi가 반환하는 response를 회의 결과에 따라 변경한다.

maong0927 commented on web/server/src/routers/comment.ts:
```diff
@@ -1,5 +1,18 @@
+  try {
+    const { body } = req;
+    const newComment = await CommentService.create(body);
+    res.status(200).json(newComment);
```
status(200) 생락해주세용!!

Commit:
refactor: :rainbow: 코드리뷰 리팩토링

팀원분들에게 받은 코드 리뷰를 기반으로 리팩토링을 진행하였다.
status(200)을 생략하였다.

namda-on reviewed (approved):
고생하셨슴돵!!!
